src/index.js

```diff
@@ -26,7 +26,25 @@ function resolveConfig(configPath) {
   }
 }
 
-function loadConfig(configPath) {
+/**
+ * Allow tool developer try to figure out if the user has specified the `lint-staged`
```

Comment:
I don't quite get what this says. Can you please re-write it so that's more clear?